scripts/gitlab/check_runtime.sh

```diff
@@ -93,16 +91,14 @@ then
 
 	# okay so now need to fetch the substrate repository and check whether spec_version or impl_version has changed there
 	SUBSTRATE_CLONE_DIR="$(mktemp -t -d substrate-XXXXXX)"
-	trap "rm -rf ${SUBSTRATE_CLONE_DIR}" INT QUIT TERM ABRT EXIT
-
+	trap 'rm -rf "${SUBSTRATE_CLONE_DIR}"' INT QUIT TERM ABRT EXIT
 
 	git clone --branch polkadot-master --depth 100 --no-tags \
-	  ${SUBSTRATE_REPO} ${SUBSTRATE_CLONE_DIR}
-
+		"${SUBSTRATE_REPO}" "${SUBSTRATE_CLONE_DIR}"
 
 	# check if there are changes to the spec|impl versions
-	git -C ${SUBSTRATE_CLONE_DIR} diff \
-		${SUBSTRATE_PREV_REF}..${SUBSTRATE_NEW_REF} ${SUBSTRATE_VERSIONS_FILE} \
+	git -C "${SUBSTRATE_CLONE_DIR}" diff \
+		"${SUBSTRATE_PREV_REF}..${SUBSTRATE_NEW_REF} ${SUBSTRATE_VERSIONS_FILE}" \
```

Comment:
not sure if git will handle this correctly as these are two arguments passed on as one